scripts/check_admin_ui_urls.py

```
@@ -0,0 +1,393 @@
+                    return UrlResult(target, False, get_error.code, str(get_error.reason or ""))
+                except urllib.error.URLError as get_error:
+                    return UrlResult(target, False, 0, f"URLError: {get_error.reason}")
+            return UrlResult(target, False, error.code, str(error.reason or ""))
+        except urllib.error.URLError as error:
+            last_detail = f"URLError: {error.reason}"
+            if attempt < max_retries - 1:
+                time.sleep(delay)
+                delay *= 2
+                continue
+            return UrlResult(target, False, 0, last_detail)
+        except Exception as error:  # Keep one unexpected site from aborting the run.
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)